api/main.py

```diff
@@ -138,6 +139,41 @@ async def post_user(
     return obj
 
 
+@app.post('/group/{group_name}', response_model=UserGroup)
+async def post_user_group(
+        group_name: str,
+        current_user: User = Security(get_user, scopes=["admin"])):
+    """Create new user group"""
+    try:
+        obj = await db.create(UserGroup(user_group=group_name))
```

Comment:
Shouldn't it be `name=`?
```suggestion
        obj = await db.create(UserGroup(name=group_name))
```

Reply:
Oh, yes. I missed it somehow. Let me fix it.